Formula/tor.rb

```diff
@@ -39,15 +39,6 @@ def install
     system "make", "install"
   end
 
-  def caveats; <<-EOS.undent
```

Comment:
> For flip sake Mike, I wasn't taking a dig at you, and I wish you'd stop reacting like I'm out to get you every time I breathe something you consider negative.

I don't think everything is a dig at me but I do find the "here's what I think but no-one listens to me now" thinking pretty tiresome when it's repeated.